notebooks/234-blip-visual-language-processing/234-blip-visual-language-processing.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>Please add doc string, type hints, and comments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Thanks @eaidova for this notebook, I wanted to see that in our repo, I have added my comments in the notebook. 

I got an error in the second cell. 
![image](https://user-images.githubusercontent.com/10940214/221939593-af61122f-e618-4ca5-918b-e095f770e0ee.png)

I think it is also interesting to show how OpenVINO is improving its performance in comparison with the original models.

Reply:
which transformers version do you have?

Reply:
Over my previous installation, I reinstalled the requirements. 

![image](https://user-images.githubusercontent.com/10940214/221995391-727b28c5-49d2-4864-96cf-2060c8359348.png)

Reply:
ok, It was introduced on 4.26.0, I'll add to notebook requirements that